changelog.md

```diff
@@ -1,5 +1,38 @@
 # Changelog
 
+
+## 0.15.0
+
+This release vastly expands printer options by using IPP-over-USB to support most AirPrint-capable printer with a USB port, in addition to switching to the modern GTK print dialog. This release also adds progress bars to give users visual feedback while a file is downloading.
+
+* Initial support for IPP / driverless printing (#2332, #2434, #2433)
+* Replace XPP with GTK-based print dialog (#2411)
+* Display download progress for file downloads (#2327, #2428, #2427, #2426, #2424, #2329)
+
+* Localization
+  *  Standardize on "drive" (versus "device") for USB export devices (#2251)
```

Comment:
Ah sorry, I meant this didn't need to be under "Localization", just make it a top-level bullet